docs/index.txt

```diff
@@ -252,6 +266,18 @@ there are elements in the array.
     arrayFieldResultSet.first();
     String firstValue = arrayFieldResultSet.getString("array_field");
 
+When inserting arrays using a prepared statement, you must convert your java array
+to a ``java.sql.Array`` by the use of ``createArrayOf()``. This function takes
+as its first argument a Crate type as described above and as its second the array
+you want to convert. You can then use ``setArray()`` to set this converted array
+using a prepared statement's argument.
+
+.. code-block:: java
+
+    PreparedStatement preparedStatement = connection.prepareStatement("insert into my_table " +
```

Comment:
In the crate project we have/had a java doctest layer to ensure examples are actually runnable. Might make sense to start using this here too?